terraform/groups/ecs-service/main.tf

```diff
@@ -19,7 +19,7 @@ terraform {
 }
 
 module "secrets" {
-  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.235"
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.241"
```

Comment:
Would we need to keep changing the ref regularly?

Reply:
only if there are fixes or major config updates.